common/scripts/build-busybox.sh

```diff
@@ -55,8 +55,23 @@ if [ $BAND == "SR" ]; then
 else
     . $TOP_DIR/../../common/config/common_config.cfg
 fi
+# toolchain
+. $TOP_DIR/../../common/scripts/common_cross_toolchain.sh
```

Comment:
Refactor the changes in file to a new file common/scripts/build-busybox-**arm**.sh

The original file common/scripts/build-busybox.sh shall be unmodified.